k8s.gcr.io/manifests/k8s-staging-build-image/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
Actually, I think manifests/... should NOT have OWNERS files - that's more privileged than merely pushing a new image.

Reply:
ref: https://github.com/kubernetes/k8s.io/pull/612, https://github.com/kubernetes/k8s.io/pull/611